Ebraheem-Mahmood/style.css

```diff
@@ -0,0 +1,149 @@
+body{
+	align-items: center;
+	font-family: Segoe UI,Frutiger,Frutiger Linotype,Dejavu Sans,Helvetica Neue,Arial,sans-serif; 
+	background-color: #f9f2f2;
+}
+
+
+
+header{
+	position: fixed;
+	top:0;
+	right: 0;
+	left: 0;
+	width: 100%;
+	height: auto;
+	border-bottom: #fff0ff 1px solid;
+	background-color: white;
+	
+	
+}
+
+#logo-side{
+	margin-left: 20%;
+	margin-top:2px ;
+	display: inline-block;
+
+}
+
+.logo-img{
+	width: 50px;
+	height: 50px;
+	display: inline-block;
+}
+
+.btn-side{
+	margin-right: 20%;
+	margin-top:20px ;
```

Comment:
Avoid using percentages (%) for margins. It's not the best practice. It is better to use pixels instead.